assets/js/components/ChargingSessionModal.vue

```diff
@@ -140,25 +168,45 @@
 </template>
 
 <script>
-import Modal from "bootstrap/js/dist/modal";
 import "@h2d2/shopicons/es/regular/checkmark";
+import { distanceUnit, distanceValue } from "../units";
 import formatter from "../mixins/formatter";
+import Modal from "bootstrap/js/dist/modal";
 import api from "../api";
-import { distanceUnit, distanceValue } from "../units";
+import store from "../store";
```

Comment:
Der Store sollte nie direkt in Unterkomponenten verwendet werden - nur in den Hauptseite (router). Von da aus dann die Props top-down runter reichen. In diesem Fall gehört der Import nach `ChargingSessions.vue` und dann über prop in den Dialog.